src/components/post-card.js

```diff
@@ -1,46 +1,35 @@
 import { Link } from 'gatsby';
-import PropTypes from 'prop-types';
 import Img from 'gatsby-image';
 import React from 'react';
 import style from './post-card.module.styl';
 import PostTag from './post-tag';
+import PropTypes from 'prop-types';
 
 const PostCard = ({ title, description, link, coverImage, tags }) => {
-
-  const tagsList = tags.map(tag => <PostTag key={tag} tag={tag} />);
+  const tagsList = tags.map((tag, i) => <PostTag key={i} tag={tag} />);
 
   return (
-    <div className={`${style.postCard} content-box`}>
-      {coverImage && (
-        <Link to={link}>
-          <Img fluid={coverImage} />
-        </Link>
-      )}
-      <div>
-        <Link className={style.postCardTitleLink} to={link}>
+    <Link to={link} className={style.postCardLink}>

```

Comment:
the outer link on postcard was removed, since gatsby complains when you nest the Link component